index.html

```diff
@@ -1,110 +1,94 @@
+
+   <link rel="stylesheet" href="./src/fonts.css">
+   <link rel="stylesheet" href="./src/style.css">
 </head>
 
-<body class="is-boxed has-animations lights-off">
-    <div class="body-wrap boxed-container">
-        <header class="site-header">
-            <div class="container">
-                <div class="site-header-inner">
-                    <div class="brand header-brand">
-                        <h1 class="m-0">
-                            <div class="lights-toggle">
-                                <input id="lights-toggle" type="checkbox" name="lights-toggle" class="switch">
-                                <label for="lights-toggle" class="text-xs">
-                                </label>
-                            </div>
-                        </h1>
-                    </div>
-                </div>
+<body>
+   <div class="container">
+    
+      <div class="toggle-container">
+         <button class="toggle" for="toggle-button" tab-index="1">
+            <div class="icon">
+               <i class="moon off"></i>
+               <i class="sun on"></i>
+            </div>
+         </button>
+      </div>
+
+      <div class="cards">
+         <div class="card">
+            <div class="left">
+               <a href="//docs.sublimetext.io">
+                  <img class="app-icon" src="./src/images/tinified/sublime.png" alt="sublime logo">
+               </a>
+            </div>
+            <div class="right">
+               <a href="//docs.sublimetext.io">
+                  <h3>Unofficial Sublime Text Documentation</h3>
+               </a>
+               <hr />
+               <div class="text">
+                  The Sublime Text Community Documentation is a community effort at
+                  documenting the <a href="https://sublimetext.com/">Sublime Text</a> editor
+                  and accompanying its <a href="https://sublimetext.com/docs/3/">official documentation</a>.
+               </div>
+            </div>
+         </div>
+
+         <div class="card">
+            <div class="left">
+               <a href="//lsp.sublimetext.io">
+                  <img class="app-icon" src="./src/images/tinified/sublime-lsp.png" alt="sublime lsp logo">
+               </a>
+            </div>
+            <div class="right">
+               <a href="//lsp.sublimetext.io">
+                  <h3>Sublime LSP Documentation</h3>
+               </a>
+               <hr />
+               <div class="text">
+                  Sublime Text's community driven implementation for LSP support.
+               </div>
+            </div>
+         </div>
+
+         <div class="card">
+            <div class="left">
+               <a href="http://discord.sublimetext.io/">
+                  <img class="app-icon" src="./src/images/tinified/discord.png" alt="discord logo">
+               </a>
+            </div>
+            <div class="right">
+               <a href="https://discord.sublimetext.io/">
```

Comment:
```suggestion
               <a href="http://discord.sublimetext.io/">
```

https doesn't work with the redirect, unfortunately.

Reply:
oops! i updated, sorry about that!